_posts/2020-12-19-fixing-apple-smart-keyboard-with-copper-tape.md

```diff
@@ -0,0 +1,74 @@
+
+I took the keyboard to Apple authorized service centre, which is run by the same company from where bought the iPad Air and Smart Keyboard originally. They first confirmed the problem is on the keyboard by connecting the keyboard to different iPad and connecting a different keyboard to my iPad.
+
+After confirming the problem is on the keyboard, they checked the warranty and realized it has expired one month back. Apparently, Apple has a replacement program for Smart Keyboard(they didn't say it was a manufacturing defect, but I wonder why else they have replacement program). But, they are not sure whether my keyboard is covered under the program. They asked me to hand over the keyboard and said they would call me once they hear back from Apple. I got a call in two days, and I was told Apple can't provide a replacement. I insisted on providing a replacement or at least repair it as it is just one month past from the warranty period. They said they can neither provide replacement and nor repair as these keyboards non-serviceable. The only option they offered me was buying a new Apple Smart Keyboard, which would cost around Rs.14,000.
+
+I did some research on the Internet and found out that many people had faced the same issue with Apple Smart Keyboard. I also found YouTube videos [1], [2] explaining how we can fix it with copper tapes or soldering.
+
+## Cause
+
+Apple Smart Keyboard uses conductive tapes to connect the Smart Connectors with keyboard electronics. Allegedly, there are two common issues with this connection. Either the connection between Smart Connectors and conductive tape is worn out(as it happened in my case) or the conductive tape itself is worn out(as shown in the YouTube video). In both cases, the keyboard wouldn't get recognized by the iPad.
+
+![Smart Keyboard Connector Pins](/img/smart-keyboard-connector-pins.jpg)
+
+![Smart Keyboard Schematic](/img/smart-keyboard-schematic.jpg)
+
+## Fix
+
+The fix is as simple as reestablishing the connection between Smart Connector and conductive tape or between the disconnected areas of the conductive tape.
+
+The YouTube videos tried best to hide the patchwork within the outer layer of the keyboard. The videos soldered a thin wired between the Smart Connector and the conductive to hide the patchwork. Despite having soldering kit and having done soldering multiple times, I am not good at soldering tiny components. I was concerned about damaging the Smart Connector pins or the keyboard shell. At the same time, I wasn't too much concerned aesthetic. So, I decided to avoid soldering and keep all the patchwork on the outer layer of the keyboard itself.
```

Comment:
Very minor typos: 

Suggested corrected version:

`The YouTube videos tried best to hide the patchwork within the outer layer of the keyboard. The videos soldered a thin wire between the Smart Connector and the conductive tape to hide the patchwork. Despite having a soldering kit and having done soldering multiple times, I am not good at soldering tiny components. I was concerned about damaging the Smart Connector pins or the keyboard shell. At the same time, I wasn't too concerned about aesthetics. So, I decided to avoid soldering and keep all the patchwork on the outer layer of the keyboard itself.
`